improver_tests/calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -332,185 +253,160 @@ def test_with_kwargs(self):
-
-        frt_orig_coord = self.historic_forecast.coord("forecast_reference_time")
-        frt_point = np.min(frt_orig_coord.points)
-        frt_coord = frt_orig_coord.copy(frt_point)
-
-        aux_coords_and_dims = [
-            (coefficient_name, 0),
-            (time_coord, None),
-            (frt_coord, None),
-            (self.historic_forecast[-1].coord("forecast_period"), None),
-            (self.x_coord, None),
-            (self.y_coord, None),
-        ]
-
-        attributes = {
-            "mosg__model_configuration": "uk_det",
-            "diagnostic_standard_name": "air_temperature",
-        }
-
-        expected = iris.cube.Cube(
-            optimised_coeffs,
-            long_name="emos_coefficients",
-            units="1",
-            dim_coords_and_dims=dim_coords_and_dims,
-            aux_coords_and_dims=aux_coords_and_dims,
-            attributes=attributes,
-        )
-
         plugin = Plugin(
             distribution=self.distribution,
-            current_cycle=self.current_cycle,
             desired_units=self.desired_units,
             predictor=predictor,
         )
-        result = plugin.create_coefficients_cube(
+        result = plugin.create_coefficients_cubelist(
             optimised_coeffs, self.historic_forecast_with_realizations
         )
-        self.assertEqual(result, expected)
-        self.assertArrayEqual(
-            result.coord("coefficient_name").points, expected_coeff_names
-        )
-
-    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_coefficients_from_mean_non_standard_units(self):
-        """Test that the expected coefficient cube is returned when the
-        historic forecast units are non-standard."""
-        expected_coeff_names = ["gamma", "delta", "alpha", "beta"]
-        self.historic_forecast.coord("forecast_period").convert_units("hours")
-        self.expected.coord("forecast_period").convert_units("hours")
-        result = self.plugin.create_coefficients_cube(
-            self.optimised_coeffs, self.historic_forecast
-        )
-        self.assertEqual(result, self.expected)
-        self.assertEqual(self.plugin.coeff_names, expected_coeff_names)
-
-    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_forecast_period_coordinate_not_present(self):
-        """Test that the coefficients cube is created correctly when the
-        forecast_period coordinate is not present within the input cube."""
-        self.expected.remove_coord("forecast_period")
-        self.expected.remove_coord("time")
-        self.historic_forecast.remove_coord("forecast_period")
-        result = self.plugin.create_coefficients_cube(
-            self.optimised_coeffs, self.historic_forecast
-        )
-        self.assertEqual(result, self.expected)
+        self.assertEqual(len(result), 4)
+        for cube in result:
```

Comment:
Yeah OK, if you want.  You shouldn't have to repeat every assertion here as you're covering the same code multiple times, but it doesn't do any huge damage.